docs/RELEASE_NOTES.md

```diff
@@ -1,6 +1,17 @@
 
 # Release notes
 
```

Comment:
This is already done in develop branch. We never push directly to master, only to develop.